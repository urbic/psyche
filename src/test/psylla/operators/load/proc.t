/ABC { dup 1 exch div } def
/ABC load ?
