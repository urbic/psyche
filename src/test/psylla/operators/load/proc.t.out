{dup 1 exch div}
